example/src/pages/App.tsx

```diff
@@ -375,31 +378,49 @@ class App extends React.Component<any, any> {
 
       // @ts-ignore
       function sendTransaction(_tx: any) {
-        return new Promise((resolve, reject) => {
-          web3.eth
-            .sendTransaction(_tx)
-            .once("transactionHash", (txHash: string) => resolve(txHash))
-            .catch((err: any) => reject(err));
-        });
+        if (providerCached === "custom-wallyconnect") {
+          return new Promise((resolve, reject) => {
+            provider
+              .request({
```

Comment:
what version of the sdk are you using? / what method is `web3.eth.sendTransaction` using under the hood? things should be working a little bit better in the lastest version, or the one up [in a PR](https://github.com/Wally-xyz/wallet-connector/pull/23) right now